### DISCUSSION TITLE
Possibility to manage Dev Portal with Microfrontend Composer

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#444] [Q&A] [answered]
### What product are you having troubles with?

Microfrontend Composer

### What Console version are you using?

12.3.0

### Description

Hi! 

At the moment, once I create the Dev Portal from the Applcations Marketplace, I have the possibility to configure the Backoffice with the Microfrontend Composer but I still need to manually modify the configmap for configuring the Dev Portal itself.

Since they are both a microlc application, is there a way to handle both with the microfrontend composer section?

If the microfrontend composer can only handle one application per project, can I switch the composer to handle the Dev Portal rather then the backoffice?

Thanks

**Comment by epessina** [comment, marked as answer]
Hi @Lucascanna ! 

At the moment, the configurator can handle only one composer (as in one micro-lc instance) per Console project.

The logic with which the instance is chosen is simply the first one it finds reading the list of microservices. Therefore, unfortunately, there is no easy way of changing the handled instance.

Please, let's track the issue in our internal product backlog so that we can address this necessity.

Thanks!
